backend/app/anti_recommenders/anti_recommender.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod 
+
+class AntiRecommender(ABC):
+    """
+    An interface for the anti-recommendation engine
+    """
+
+    @abstractmethod 
+    def generate_anti_recommendations(self) -> None:
```

Comment:
Unclear why the return type for this would be None.

Reply:
Are you using mypy and ruff?

Reply:
Yes, I am using mypy and ruff now.

Reply:
I am yet to fix all of mypy's suggestions, but this return type error has been corrected.